src/sentry/static/sentry/app/components/stackedBarChart.jsx

```diff
@@ -100,7 +101,7 @@ class StackedBarChart extends React.Component {
     }
   }
 
-  shouldComponentUpdate(nextProps, nextState) {
+  shouldComponentUpdate(nextProps, _nextState) {
```

Comment:
i got stuck w/ lint violations (did we not codemod/fix when we added lint rules?)